django_admin_reset/templates/admin/change_user.html

```diff
@@ -3,7 +3,7 @@
 
 {% block form_top %}
 <p>
-<a href="{% url 'admin:password_reset_url' id=original.pk %}">
+<a href="{% url 'admin:password_reset_url' id=original.pk|lower %}">
```

Comment:
Why are you converting to lowercase? This will break, if a user ID has uppercase letters.

Reply:
a mistake for somereson